tools/build-utils.js

```diff
@@ -236,9 +236,10 @@ exports.transpile = (doForce, decl) => {
 	}
 
 	// sucrase doesn't copy JSON over, so we'll have to do it ourselves
-	copyOverDataJSON('bss-factory-sets.json');
+	copyOverDataJSON('random-sets.json');
 	copyOverDataJSON('cap-1v1-sets.json');
```

Comment:
```suggestion
	copyOverDataJSON('mods/gen8/cap-1v1-sets.json');
```
can you also move that file into the gen 8 mod?